### ISSUE TITLE
GUI: Motors perspective update too slow

### ISSUE THREAD

**Issue by IsabellaRey** [ISISComputingGroup/IBEX#1613]
The motors perspective can be extremely slow (minutes) at picking up motor IOCs running. This was seen while running manual system tests on NDXDEMO, looking at the MCLEN motors

**Comment by AdrianPotter** [comment]
Believe fixed with #1614

**Comment by IsabellaRey** [comment]
I have checked this running MCLEN IOCs (up to 3 IOCs at the same time) in DEVSIM mode.

There is an improvement in that it seems to take little time to see some sort of indication that something is happening (as long as the perspective hasn't been open for too long, see below). However, it still takes a long time for the table to fully update:

* when starting 3 IOCs at the same time, I have timed ~5min30s, ~7min30s and >9min for the table to fully update.

The total waiting time seem to increase the longer I keep the perspective open. Even when starting a single IOC, at first it seems to populate quite quickly, but if the perspective has been open for some time, it takes longer (i.e. with a single MCLEN running, I left it for the entire standup and only one cell was showing the motor name). After that, even when starting a single IOC there was no indication at all on the table that anything was running.
